Libs/MRML/Core/vtkMRMLAbstractViewNode.h

```diff
@@ -110,9 +110,19 @@ class VTK_MRML_EXPORT vtkMRMLAbstractViewNode
   /// \sa GetVisibility()
   /// \sa IsViewVisibleInLayout()
   /// \sa vtkMRMLLayoutNode::SetViewArrangement()
+//TODO: Remove
```

Comment:
Are you sure we can remove this? Why?

Reply:
We can indicate the same status with the parent layout reference, can't we?